sources/packages/web/src/components/common/StatusInfoEnrollment.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <status-info-label :status="enrollmentStatus.status">{{
+    enrollmentStatus.header
+  }}</status-info-label>
+</template>
+
+<script lang="ts">
+import { PropType, computed, defineComponent } from "vue";
+import { StatusInfo, COEStatus } from "@/types";
+
+export interface EnrollmentStatusInfo {
```

Comment:
i feel enrolment right? not enrollment.

Reply:
enrolment (British way) and enrollment both are right. We use enrolment as confirmed by @hellolynn-tbtb